tcms/testplans/static/testplans/js/get.js

```diff
@@ -686,6 +686,18 @@ function toolbarEvents(testPlanId, permissions) {
 
          window.location.assign(`/cases/clone?case=${selectedCases.join('&case=')}`);
     });
+
+    $('#testplan-toolbar-newrun').click(function(ev) {
+        let selectedTestCases = getSelectedTestCases();
+
+        if (!selectedTestCases.length) {
+            alert($('#test_plan_pk').data('trans-no-confirmed-testcases-selected'));
+            return;
+        }
```

Comment:
Also needs to check statuses of all the selected test cases!

Reply:
Aren`t we discussed that already? We show a warning in create new run view, so the user is informed about that, and need to go back and reselect, also we should the statuses at create new run view.

Reply:
I don't remember but it is a relatively easy check and it can be added. Also the fact that there is a message in the HTML template suggests that we were thinking to add a simple check in the UI before potentially sending thousands of selected cases to the backend for examination.